public/app/features/alerting/unified/components/silences/MatchedSilencedRules.tsx

```diff
@@ -75,24 +79,30 @@ function useColumns(): MatchedRulesTableColumnProps[] {
     {
       id: 'state',
       label: 'State',
-      renderCell: function renderStateTag({ data: { matchedRule } }) {
-        return <RuleState rule={matchedRule} isCreating={false} isDeleting={false} />;
+      renderCell: function renderStateTag({ data: { matchedInstance } }) {
+        return <AlertStateTag state={matchedInstance.state} />;
       },
       size: '160px',
     },
     {
-      id: 'name',
-      label: 'Name',
-      renderCell: function renderName({ data: { matchedRule } }) {
-        return matchedRule.name;
+      id: 'labels',
+      label: 'Labels',
+      renderCell: function renderName({ data: { matchedInstance } }) {
+        return <AlertLabels labels={matchedInstance.labels} />;
       },
       size: '250px',
     },
     {
-      id: 'summary',
-      label: 'Summary',
-      renderCell: function renderSummary({ data: { matchedRule } }) {
-        return matchedRule.annotations[Annotation.summary] ?? '';
+      id: 'created',
+      label: 'Created',
+      renderCell: function renderSummary({ data: { matchedInstance } }) {
+        return (
+          <>
+            {matchedInstance.activeAt.startsWith('0001')
+              ? '-'
+              : matchedInstance.activeAt.substr(0, 19).replace('T', ' ')}
```

Comment:
Not introduced in this PR but a very weird way to format timestamps, can we go with `dateTime(matchedInstance.activeAt).format('YYYY-MM-DD HH:mm:ss')` from `@grafana/data` instead?